plugins/view-transitions/includes/theme.php

```diff
@@ -355,3 +355,25 @@ function plvt_load_view_transitions(): void {
 	wp_add_inline_script( 'plvt-view-transitions', $init_script );
 	wp_enqueue_script( 'plvt-view-transitions' );
 }
+
+/**
+ * Outputs the necessary CSS styles for view transitions.
+ *
+ * This function is responsible for printing the required inline styles
+ * to enable or enhance view transitions within the theme or plugin.
+ * It should be hooked to an appropriate action to ensure the styles
+ * are included in the page output.
+ *
+ * @since n.e.x.t
+ */
+function plvt_print_view_transitions_admin_style(): void {
+	$options = plvt_get_stored_setting_value();
+	if ( ! isset( $options['enable_admin_transitions'] ) || true !== $options['enable_admin_transitions'] ) {
+		return;
+	}
+	?>
+<style>
+	@view-transition { navigation: auto; }
```

Comment:
I found a way to make the transitions for the admin menu _a lot_ smoother than the default. Try this:
```suggestion
	@view-transition { navigation: auto; }
	#adminmenu > .menu-top { view-transition-name: attr(id type(<custom-ident>), none); }
```

It'll use the ID for every top-level menu item for the view transition name. This way, each item morphs between the two views you navigate between. I think that makes it look much better than just fading the entire screen into each other. You can especially tell if you click on a top-level menu item that is lower in the admin menu than the currently expanded top-level menu item.

Technical note: The `attr()` function is baseline in CSS, see https://developer.mozilla.org/en-US/docs/Web/CSS/attr. It takes the value of an attribute and uses it for the CSS property, making the implementation of this really simple, since the view transition name needs to be unique for each individual menu item.

Reply:
It looks awesome.